docs/genai/api/java.md

```diff
@@ -0,0 +1,595 @@
+---
+title: Java API
+description: Java API reference for ONNX Runtime generate() API
+has_children: false
+parent: API docs
+grand_parent: Generate API (Preview)
+nav_order: 1
+---
+
+# ONNX Runtime generate() Java API
+
+_Note: this API is in preview and is subject to change._
+
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Install and import
+The Java API is delivered by the ai.onnxruntime.genai Java package. Package publication is pending. To build the package from source, see the [installation guide](../howto/install.md).
```

Comment:
This should actually be the build-from-source guide

Reply:
fixed.